docs/Displaying a User Interface/Text Images and Buttons/index.md

```diff
@@ -10,6 +10,7 @@ The @![iOS]`SDLScreenManager`!@ @![android, javaSE, javaEE]`ScreenManager`!@ is
 | textField2 | The text displayed on the second display line of a multi-line display |
 | textField3 | The text displayed on the third display line of a multi-line display |
 | textField4 | The text displayed on the bottom display line of a multi-line display |
+| title | The title of the new template that will be displayed |
```

Comment:
```suggestion
| title | The title of the displayed template |
```

Reply:
I don't like the fact that iOS and Android have different names for these parameters. We should align them.